client/src/common/components/Breadcrumbs/index.jsx

```diff
@@ -2,14 +2,20 @@ import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import classNames from 'classnames';
+import { injectIntl } from 'react-intl';
+import _tail from 'lodash/tail';
 
 import { ChevronRight } from 'assets/images/icons';
+import { IntlPropType } from 'common/constants/propTypes';
 
-const Breadcrumbs = ({ breadcrumbs, isGuest }) => {
+const Breadcrumbs = ({ breadcrumbs, isGuest, intl: { formatMessage } }) => {
   if (breadcrumbs.length <= 1) {
     return null;
   }
 
+  const restOfTheBreadcrumbs = _tail(breadcrumbs);
+  const firstBreadcrumb = breadcrumbs[0];
```

Comment:
why not just:
`const [firstBreadcrumb, ...restOfTheBreadcrumbs] = breadcrumbs;`
?